EIPS/draft_hardware_and_bandwidth_recs.md

```diff
@@ -0,0 +1,147 @@
+
+### RAM
+
+RAM/memory is dominated by state cache. As of January 2025, it is possible to run a full node with 16GB of RAM, however this has been known to not work with all combinations of EL and CL clients in the past.
+We therefore recommend 32GB.
+
+### Bandwidth
+
+- Statista states that as of January 2024:
+  - The global average download for broadband is 92 Mbps and the global average upload is 43 Mpbs.
+  - The global average download for mobile is 50 Mbps and 11 Mbps
+- GSMA report showing the state of mobile internet connectivity in 2024 shows that:
+  - The mobile upload speeds in Higher Income Countries (HIC) is about 18 Mbps
+  - The global average mobile download is 48 Mbps.
+
+Although most users will be using fixed broadband as opposed to mobile broadband. We include the mobile broadband figures as a way to approximate lower bounds.
+
+#### Block builders
+
+For validators that want to build blocks locally, we recommend global bandwidth figures inline with the global average for fixed broadband: 100 Mbps download and 50 Mbps upload.
+
+*If a block builder does not have the required bandwidth, we recommend that they build a block that is partially full and or one that includes less blobs.*
+
+#### Attesters
+
+For attesters, we recommend: 50 Mbps download and 25 Mbps upload.
+
+The minimum value recorded to work as of January 2025 has been ~15 Mbps. This was recorded in simulations done by Eth panda ops where 40% of the network had Gigabit connections and the other 60% had 15 Mbps upload.
+
+The minimum value could be lower than this and is highly dependent on your network topology.
+For example, if you are connected to peers whom have gigabit connections, but you yourself have subpar bandwidth, it may be sufficient for you to only send your data to the first peer and in the time it takes you to send data to your mesh, the first peer might have seeded the network.
+
+The 25 Mbps was chosen as a buffer to account for these miscellaneous factors that are harder to predict for.
+
+#### Validators using mev-boost
+
+The recommendation for validators using mev-boost is the same as the recommendations for attesters.
+
+#### Full nodes
+
+For full nodes, whom want to follow the tip of the chain, we recommend 50 Mbps download and 15 Mbps upload.
+
+Full nodes currently participate in sampling, however since they are not latency sensitive like attesters and block builders, we can lower their upload bandwidth requirement.
+
+The download requirement is kept the same, so that they are able to follow the chain at the same speed as an attester.
+
+Note: A full node will still be able to operate at significantly lower bandwidth requirements, however depending on the variance they may be at least a slot behind the chain.
+
+## Backwards Compatibility
+
+This EIP is informational and requires no protocol changes. We recommend that future EIPs include an assessment of their impact on these hardware requirements.
```

Comment:
Would the recommendation here require another EIP if this gets finalized?

In any case, I don't think it belongs here since this section is explicitly about whether the EIP breaks backwards compatibility